### PR TITLE
Add SilvermanAGutData

### PR DESCRIPTION
**Pull request by TuomasBorman**
https://github.com/microbiome/mia/pull/289

### CODE REVIEW

File: R/data.R [hunk 1]
```diff
@@ -83,4 +83,39 @@ NULL
 #'
 NULL
 
-
+#' @title SilvermanAGutData
```
Comment (antagomir) [inline R/data.R:86]: We can omit "@title" - the first row is interpreted as title. This is how we do it most of the time.

File: R/data.R [hunk 2]
```diff
@@ -83,4 +83,39 @@ NULL
...
+#' @description The SilvermanAGutData dataset contains 16S rRNA gene based high-throughput
```
Comment (antagomir) [inline R/data.R:87]: Same here, we can omit @description tag. The text that comes after the first row (title) is interpreted as description.

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
Would be good to make a brief check in OMA,mia, miaViz, miaTime if they used this data in examples or vignettes, and change accordingly..

File: R/data.R (hunk 1 above)
Comment (TuomasBorman) [reply R/data.R:86]: I will change those, but I just followed other formatting of other datsets in miaTime (tehy ahve title and description

File: R/data.R (hunk 2 above)
Comment (antagomir) [reply R/data.R:87]: Ah. Ok let's use the same everywhere. Easier to do same as the already existing. If we change later we can change all at once.

**Comment by TuomasBorman** [comment]
Yes, it is

**Pull request merged**